### PR TITLE
Feat/properties on aqara smartplug

### PR DESCRIPTION
**Pull request by mrsteakhouse**
Hi, this adds support for the energy counter attribute in a seMetering cluster.
This also depends on this: https://github.com/WebThingsIO/zcl-id/pull/2.

### CODE REVIEW

**Comment by mrsteakhouse** [comment]
I tested this with my setup during the last weekend. The property's unit is off, since the gateway does not know about kW. Or i'm  blind enough to overlook things.

**Comment by mrsteakhouse** [comment]
@chas-iot I would be happy if you can find some time to test this as well.

It would be a good idea to add a testing framework and implement regression tests somewhere in the future.

**Comment by chas-iot** [comment]
> @chas-iot I would be happy if you can find some time to test this as well.

Please see https://github.com/WebThingsIO/serial-prober-node/issues/11 for why I can't test at the moment ☹️

File: zb-classifier.js [hunk 1]
```diff
@@ -897,6 +897,63 @@ class ZigbeeClassifier {
...
+      // new attribute definitions are available
+      1000                            // defaultValue
+    );
+    this.addProperty(
+      node,                           // device
+      'counter',                      // name
+      {// property description
+        // even though the content is not instantaneous power, this is the only
+        // property type with watt as unit
+        '@type': 'InstantaneousPowerProperty',
```
Comment (mrstegeman) [inline zb-classifier.js:941]: Please remove the comment and the `@type` field. It will work just fine without it.

**Review by mrstegeman — CHANGES_REQUESTED** [review]

**Review by mrstegeman — APPROVED** [review]

**Pull request merged**